### PR TITLE
Updated surrender journey 

### PR DESCRIPTION
**Pull request by gonogo**
Update surrender journey to include feedback from crit: radically simplified interface, also handles scenarios where user hasn't yet submitted emissions, and where they have already surrendered the required amount

### CODE REVIEW

**Comment by gonogo** [comment]
There's probably a simpler way of expressing the logic here, and it could be done more elegantly, but it just about works.

File: app/views/account/surrender-allowance/surrender-amount.html [hunk 1]
```diff
@@ -10,66 +10,76 @@ <h1 class="govuk-heading-xl">
...
+                    {% else %}
+                       <p>You need to surrender <strong>{{ data.etsSubmitEmmissions.total | formatNumber }}</strong> units to cover your emissions for 2018.</p> 
+                       <ul class="govuk-list govuk-list--bullet">
+                          <li>You need to do this before the deadline or [consequences]</li>
+                          <li>You may also choose to surrender more than the required amount. </li>
+                          <li>You can do this in multiple transactions.</li>
+                       </ul>
+                    {% endif %}
+                {% else %}
+                    <p>You have not yet declared you emissions but you can still surrender allowance.</p>
```
Comment (igloosi) [inline app/views/account/surrender-allowance/surrender-amount.html]: your

**Comment by igloosi** [comment]
also fix unit calculations for other scenarios

File: app/views/account/surrender-allowance/surrender-amount.html (hunk 1 above)
Comment (igloosi) [reply app/views/account/surrender-allowance/surrender-amount.html]: fixed

**Pull request merged**